plan-for-extension-exercise/src/planForExtensionExercise/Shape.java

```diff
@@ -0,0 +1,9 @@
+package planForExtensionExercise;
+
+public abstract class Shape {
```

Comment:
Purely for the fun of it, will you please humor me by changing Shape to an interface?